content/livesync/connector/index.textile

```diff
@@ -87,36 +97,37 @@ ADBC_NODES_TABLE_AUTO_CREATE=true
 The @docker-compose.yml@ will load this configuration into the @adbc@ container as environment variables.
 
 <aside data-type='important'>
-<p>If using your own API key, ensure that it has the @publish@ "capability":/auth/capabilities on all channels that you intend to publish to.</p>
+<p>If using an API key from a provider other than Ably, verify it has the @publish@ "capability":/auth/capabilities on all channels you intend to publish to. </p>
 </aside>
 
-Run @docker compose up@ to build and run the containers:
+Run @docker compose up@ to build and run the Docker containers:
 
 ```[sh]
 docker compose up --build
 ```
 
-To check everything is running correctly, ping the "health endpoint":/livesync/connector/monitor of the Database Connector:
+Ping the database connector "health endpoint":/livesync/connector/monitor to verify everything works correctly.
 
 ```[sh]
 curl localhost:2259/health
 ```
 
-The response should be similar to the following:
+The response should look similar to the following:
 
 ```[json]
 {"ably":{"status":"up"},"nodes_table":{"status":"up"},"outbox_table":{"status":"up"},"postgres":{"status":"up"}}
 ```
+h3(#varify-docker). Varify connection using Docker 
 
-With a PostgreSQL database and Database Connector instance running, the next step is to write a record to the outbox table and confirm it has been published over Ably:
+Varify your connection to the Ably network now you have a PostgreSQL database and Database Connector instance running. The following examples show how to write a record to the outbox table and confirm its publication. 
 
-Open a terminal and subscribe to a "channel":/channels named @foo@ using "Server-Sent Events":/protocols/sse
+Subscribe to a "channel":/channels named @foo@ using "Server-Sent Events:":/protocols/sse
```

Comment:
Is it important to tell them what to name their channel?
I realise it's used elsewhere on the page. I'm not sure.